core/tx_pool.go

```diff
@@ -55,6 +56,25 @@ var (
 	evictionInterval     = time.Minute   // Time interval to check for evictable transactions
 )
 
+var (
+	// Metrics for the pending pool
+	pendingDiscardMeter    = metrics.NewMeter("txpool/pending/discard") 
+	pendingReplaceMeter    = metrics.NewMeter("txpool/pending/replace")
+	pendingRLMeter         = metrics.NewMeter("txpool/pending/ratelimit") // Dropped due to rate limiting
+	pendingNofundsMeter    = metrics.NewMeter("txpool/pending/nofunds")   // Dropped due to out-of-funds
+	pendingTooOldMeter     = metrics.NewMeter("txpool/pending/oldnonce")   // Dropped due to old nonce
+
+	// Metrics for the queued pool
+	queuedDiscardMeter     = metrics.NewMeter("txpool/queued/discard")
+	queuedReplaceMeter     = metrics.NewMeter("txpool/queued/replace")
+	queuedRLMeter          = metrics.NewMeter("txpool/queued/ratelimit")  // Dropped due to rate limiting
+	queuedNofundsMeter     = metrics.NewMeter("txpool/queued/nofunds")    // Dropped due to out-of-funds
+	queuedTooOldMeter      = metrics.NewMeter("txpool/queued/oldnonce")   // Dropped due to old nonce
```

Comment:
This was used instead of the pending by accident, but not needed. Please see https://github.com/ethereum/go-ethereum/pull/3157/files#r85916444